src/index.ts

```
@@ -1,62 +1,46 @@
-    client.once(event.name, (...args) => event.execute(...args));
-  } else {
-    client.on(event.name, (...args) => event.execute(...args));
-  }
-});
-
-client.login(env.DISCORD_TOKEN);
+import { env } from '@/env';
+import { events } from '@/events';
+import logger from '@/lib/logger';
+import { beginStatusUpdates } from '@/utils/status';
+import { Client } from 'discord.js-selfbot-v13';
```

ESLint

[error] Unable to resolve path to module 'discord.js-selfbot-v13'.

(import-x/no-unresolved)